### ISSUE TITLE
​​Diagnosis of Diabetes using Support Vector Machines

### ISSUE THREAD

**Issue by Inyrkz** [section-engineering-education/engineering-education#1533]
**NOTE: We tend to stray away or tend not to publish reviews/comparisons of commercial product offerings.**

### Brief Summary: 
Describe the what, why, and how of your content idea in 2-5 sentences.

In this article, you will learn how to diagnose if a patient has diabetes based on his medical records. We will use the Support Vector Machine (SVM) Algorithm from Sci-kit Learn to build our Machine Learning model. After reading this article, you will be able to solve any classification problem using the support vector machine algorithm from sklearn.

### Key Takeaways: 
What are the 3-5 most important things the reader should understand or be able to do after reading this article?

After reading this article, the reader should be able to:

1. import dataset and do data preprocessing,
2. split the dataset into training and test set
3. build an SVM model using the linear, poly, RBF, and sigmoid kernel (Hyper-Parameter Optimization), 
4. make a diagnosis on new patient,
5. assess the model's performance using accuracy score, precision, recall, and f1-score.

### References: 
Please list links to any published content/research that you intend to use to support/guide this article. (If none, please indicate 'N/A'.)

- Coursera Guided Project https://www.coursera.org/projects/medical-diagnosis-support-vector-machines
- Sklearn documentation https://scikit-learn.org/stable/modules/svm.html#svm-kernels

**Comment by hectorkambow** [comment]
Thank you for your topic suggestion. Per the recent message posted in our EngEd Slack channel, we are pausing acceptance of new articles until further notice while we work through the backlog of submitted drafts. We will notify you when we are ready to accept new topic suggestions. Thank you for your patience.
